storage/local_file_system.go

```diff
@@ -34,7 +36,8 @@ func NewLocalFileSystemStorageProvider(path string) (*LocalFileSystemStorageProv
 
 func (p *LocalFileSystemStorageProvider) ListObjects(prefix string) ([]*Object, error) {
```

Comment:
We should not modify ListObjects, all ListObjects funcs are called after `storageProviderObj, err := store.GetStorageProviderObj()`, the subpath should be handled inside GetStorageProviderObj()